Packs/Code42/ReleaseNotes/6_1_0.md

```diff
@@ -0,0 +1,15 @@
+#### Playbooks
+
+##### Code42 Exfiltration Playbook
+
+- Updated the Code42 Exfiltration Playbook playbook to enable use of the new argument to the "Code42 File Download" playbook.
+
+##### Code42 File Download
+
+- Updated the Code42 File Download playbook to enable use of the new download file by xfc id command.
+
+#### Integrations
+
+##### Code42
+- Updated the Docker image to: *demisto/py42:1.0.0.5470953*.
```

Comment:
The release note entry does not start with an approved prefix. Please remove the leading `- `.